src/large_pages/node_large_page.cc

```diff
@@ -282,20 +294,44 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+// Functions in this class must always be inlined because they must end up in
+// the `lpstub` section rather than the `.text` section.
+class MemoryMapPointer {
+ public:
+  FORCE_INLINE explicit MemoryMapPointer() {}
```

Comment:
```suggestion
  FORCE_INLINE MemoryMapPointer() {}
```
`explicit` is only important for single-arg constructors because of implicit conversion.